cobra_test.go

```diff
@@ -916,15 +916,14 @@ func TestRootUnknownCommand(t *testing.T) {
 
 func TestRootUnknownCommandSilenced(t *testing.T) {
 	r := noRRSetupTestSilenced("bogus")
-	s := "Run 'cobra-test --help' for usage.\n"
 
 	if r.Output != "" {
-		t.Errorf("Unexpected response.\nExpecting to be: \n\"\"\n Got:\n %q\n", s, r.Output)
```

Comment:
I think, there was missed verb and this line should be so: `t.Errorf("Unexpected response.\nExpecting to be: %q\n\"\"\n Got:\n %q\n", s, r.Output)`